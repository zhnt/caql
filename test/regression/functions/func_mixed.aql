print("Before function definition")

function greet() {
    print("Hello from greet function!")
}

print("After function definition")
greet()
print("After function call")
