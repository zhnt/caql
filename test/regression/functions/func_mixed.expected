Before function definition
After function definition
Hello from greet function!
After function call

